_posts/2025-02-14-this-month-in-servo.md

```diff
@@ -0,0 +1,465 @@
+-https://github.com/servo/servo/pull/34798	(@mrobinson, #34798)	script: Eliminate code duplication in the task queue (#34798)
++https://github.com/servo/servo/pull/34793	(@nicoburns, #34793)	Upgrade rustc to 1.83 (#34793)
+    upgrade
++https://github.com/servo/servo/pull/34780	(@wusyong, #34780)	chore: remove `WindowMethods::rendering_context` (#34780)
+    embedding
+>>> 2025-01-03T06:16:08Z
++https://github.com/servo/servo/pull/34813	(@wusyong, #34813)	webxr: create glwindow with Rc window and without rendering context (#34813)
+    embedding; RenderingContext trait
++https://github.com/servo/servo/pull/34810	(@simonwuelker, #34810)	Support syntax highlighting of arguments in the devtools console (#34810)
+    devtools
+-https://github.com/servo/servo/pull/34818	(@dependabot[bot], @dependabot[bot], #34818)	build(deps): bump tracing-perfetto from 0.1.4 to 0.1.5 (#34818)
+-https://github.com/servo/servo/pull/34817	(@dependabot[bot], @dependabot[bot], #34817)	build(deps): bump syn from 2.0.93 to 2.0.94 (#34817)
+-https://github.com/servo/servo/pull/34795	(@sagudev, #34795)	chore: Update wgpu (#34795)
+>>> 2025-01-04T06:03:22Z
++https://github.com/servo/servo/pull/34803	(@simonwuelker, #34803)	Fix IS_IN_SHADOW_TREE flag for descendants after Node::remove call (#34803)
+    shadowdom
+-https://github.com/servo/servo/pull/34832	(@mrobinson, #34832)	script: Expose node helpers as `NodeTraits` and give more descriptive names (#34832)
+-https://github.com/servo/servo/pull/34824	(@mrobinson, #34824)	Elide lifetimes where possible after rustup (#34824)
+-https://github.com/servo/servo/pull/34831	(@dependabot[bot], @dependabot[bot], #34831)	build(deps): bump cc from 1.2.6 to 1.2.7 (#34831)
+-https://github.com/servo/servo/pull/34830	(@dependabot[bot], @dependabot[bot], #34830)	build(deps): bump taffy from 0.7.1 to 0.7.2 (#34830)
+-https://github.com/servo/servo/pull/34829	(@dependabot[bot], @dependabot[bot], #34829)	build(deps): bump tempfile from 3.14.0 to 3.15.0 (#34829)
+-https://github.com/servo/servo/pull/34828	(@dependabot[bot], @dependabot[bot], #34828)	build(deps): bump winnow from 0.6.21 to 0.6.22 (#34828)
+-https://github.com/servo/servo/pull/34826	(@simonwuelker, #34826)	Upgrade lib to 0.20 to fix RUSTSEC-2024-0429 (#34826)
++https://github.com/servo/servo/pull/34808	(@git@pews.dev, @mrobinson, #34808)	fix: add source browsing context to `Request` and HTTP credentials prompt (#34808)
+    embedding
+-https://github.com/servo/servo/pull/34815	(@simonwuelker, #34815)	Fix building with feature=tracing (#34815)
+>>> 2025-01-05T06:03:55Z
+-https://github.com/servo/servo/pull/34837	(@servo-wpt-sync, #34837)	Update web-platform-tests to revision b'c72ee0c082d350b8b44f7a6bac1f028752b13527' (#34837)
+-https://github.com/servo/servo/pull/34787	(@willypuzzle, #34787)	Document's current script should not be updated when executing script elements inside shadow trees (#34787)
```

Comment:
This is part of the shadowdom.work.

Reply:
fixed in c5cc5ed58bdb1cf750cedd59769a5d5a0dcc7b64